jenkins/databricks/create.py

```diff
@@ -34,14 +34,17 @@ def main():
   worker_type = 'g4dn.xlarge'
   driver_type = 'g4dn.xlarge'
   cloud_provider = 'aws'
+  # comma seperated init scritps, e.g., dbfs:/foo,dbfs:/bar,...
+  init_scripts = ''
+
 
   try:
-      opts, args = getopt.getopt(sys.argv[1:], 'hw:t:k:n:i:r:o:d:e:s:',
+      opts, args = getopt.getopt(sys.argv[1:], 'hw:t:k:n:i:r:o:d:e:s:f:',
                                  ['workspace=', 'token=', 'sshkey=', 'clustername=', 'idletime=',
-                                     'runtime=', 'workertype=', 'drivertype=', 'numworkers=', 'cloudprovider='])
+                                     'runtime=', 'workertype=', 'drivertype=', 'numworkers=', 'cloudprovider=, initscripts='])
```

Comment:
'cloudprovider=, initscripts='
Should it be 'cloudprovider=', 'initscripts='?

Reply:
fixed